dbtwiz/admin/cleanup.py

```diff
@@ -0,0 +1,112 @@
+from dbtwiz.auth import ensure_auth
+from dbtwiz.manifest import Manifest
+from dbtwiz.target import Target
+from dbtwiz.logging import info, error
+
+from pathlib import Path
+from google.cloud import bigquery
+
+
+def cleanup_development_dataset(force_delete: bool) -> None:
```

Comment:
Is `cleanup` a good term to use, or should it be `clean`, `clean_up`?
Syntactically, if using `cleanup` then I think it should come at the end: `development_dataset_cleanup`

Reply:
Changed prefix to `handle_` since it handles both cases (list and delete.)

Reply:
Perhaps later we can refactor common code into a separate method and create two different commands.